compiler/frontend/spark/src/main/java/org/apache/nemo/compiler/frontend/spark/transform/LocalTextFileTransform.java

```diff
@@ -31,7 +31,7 @@
  *
  * @param <I> input type.
  */
-public final class LocalTextFileTransform<I> extends NoWatermarkEmitTransform<I, String> {
+public final class LocalTextFileTransform<I extends Serializable> extends NoWatermarkEmitTransform<I, String> {
```

Comment:
Unfortunately, this doesn't work when processing non-Serializable data elements.
Revert this change, and instead add a TODO to https://issues.apache.org/jira/browse/NEMO-431